stock_picking_invoicing/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,9 @@
+* Lorenzo Battistini <lorenzo.battistini@agilebg.com>
+* Leonardo Pistone <leonardo.pistone@agilebg.com>
+* Daniel Sadamo <daniel.sadamo@kmee.com.br>
+* Alex Comba <alex.comba@agilebg.com>
+* Florent THOMAS <florent.thomas@mind-and-go.com>
+* François Honoré <francois.honore@acsone.eu>
+* Magno Costa <magno.costa@akretion.com.br>
+* Luis Felipe Mileo <mileo@kmee.com.br>
+* Radovan Skolnik <radovan@skolnik.info>
```

Comment:
@mbcosta not sure I deserve to be left here as I didn't finish the migration.

Reply:
let's say now you have some pressure to become an OCA hero with your future contributions ;-)

Reply:
> let's say now you have some pressure to become an OCA hero with your future contributions ;-)

I'll do my best...

Reply:
Hi @Rad0van , I don't see problems if you want to be a contributor. For a while we thought this module was just used in Brazil, is better have people from other countries using, testing and giving feedback. If you have problems with the module and need to create an Issue or PR you can mark me.